src/utils/tracking.ts

```diff
@@ -14,35 +11,26 @@ export interface UtmParams {
   name?: string
 }
 
-export const setTrackingId = (): string => {
-  const id = uuid()
-  logInDev('[ANALYTICS IDENTIFY] ', id)
-  const castedWindow = window as any
-  if (castedWindow && castedWindow.analytics) {
-    castedWindow.analytics.identify(id)
-  }
-
-  return id
-}
-
 export const getUtmParamsFromCookie = (): UtmParams | undefined => {
   const params = cookie.getItem('utm-params')
   return params ? JSON.parse(params) : undefined
 }
 
-export const trackEvent = (
-  eventName: string,
-  properties?: { [key: string]: any },
-  options?: { [key: string]: any },
-) => {
-  const event = [
-    eventName,
-    merge(properties || {}, { context: { ...getUtmParamsFromCookie() } }),
-    options,
-  ]
-  logInDev('[ANALYTICS TRACK] ', event)
-  const castedWindow = window as any
-  if (castedWindow && castedWindow.analytics) {
-    castedWindow.analytics.track(...event)
-  }
+export enum CustomEvents {
```

Comment:
i don't really like the naming `Custom`, can we make it something more like `ApplicationSpecific...` or so?

Reply:
Sure!